### PR TITLE
feat: navbar

### PR DESCRIPTION
**Pull request by csosamorales**
- ~note: need to change svg update file for svg icon~[DONE]

### CODE REVIEW

**Comment by zoruka** [comment]
We might take care of mobile responsiveness in near future I think, maybe we could start working that in the current feature developments.

**Comment by jsonsivar** [comment]
[sc-712]

File: ui/src/components/core/icon/icon-library.tsx [hunk 1]
```diff
@@ -5,23 +5,32 @@ import { AiOutlineCheck } from '@react-icons/all-files/ai/AiOutlineCheck';
...
 import { IoCheckmarkCircleSharp } from '@react-icons/all-files/io5/IoCheckmarkCircleSharp';
 import { AiOutlineTwitter } from '@react-icons/all-files/ai/AiOutlineTwitter';
-import { ErrorIcon } from './custom/error';
 import { IoClose } from '@react-icons/all-files/io5/IoClose';
 import { AiFillCheckCircle } from '@react-icons/all-files/ai/AiFillCheckCircle';
 import { BiGitBranch } from '@react-icons/all-files/bi/BiGitBranch';
 
 export const IconLibrary = Object.freeze({
   back: IoArrowBackCircleSharp,
+  betaTag: BetaTag,
```
Comment (zoruka) [inline ui/src/components/core/icon/icon-library.tsx:24]: I think we should keep pattern of naming with dashes or camel case

**Review by zoruka — APPROVED** [review]

**Comment by csosamorales** [comment]
@jsonsivar sorry, it's done also that

**Review by jsonsivar — APPROVED** [review]

**Pull request merged**